godot-core/src/builtin/math.rs

```diff
@@ -119,3 +121,9 @@ pub fn cubic_interpolate_in_time(
     let b2 = lerp(a2, a3, if post_t == 0.0 { 1.0 } else { t / post_t });
     lerp(b1, b2, if to_t == 0.0 { 0.5 } else { t / to_t })
 }
+
+pub fn lerp_angle(from: f32, to: f32, weight: f32) -> f32 {
+    let difference = (to - from) % TAU;
+    let distance = (2.0 * difference) % TAU - difference;
+    from + distance * weight
+}
```

Comment:
Is this wrapping correctly?
Is the output in `[0, TAU]` or `[-PI, PI]` range or something else?

Some docs would be nice 🙂

Reply:
i just copied the code from godot, so i didn't super look into it closely.
https://github.com/godotengine/godot/blob/master/core/math/math_funcs.h#L389